src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerStepExecution.java

```diff
@@ -61,15 +61,15 @@ public boolean start() throws Exception {
         getContext().newBodyInvoker()
                 .withContext(BodyInvoker
                         .mergeLauncherDecorators(getContext().get(LauncherDecorator.class), decorator))
-                .withCallback(new ContainerExecCallback(decorator))
+                .withCallback(new ContainerExecCallback())
                 .start();
         return false;
     }
 
     @Override
     public void stop(Throwable cause) throws Exception {
         LOGGER.log(Level.FINE, "Stopping container step.");
-        closeQuietly(client, decorator);
```

Comment:
This is likely to be called on interruption.

What happens to the proc in this case? Is it handled by Jenkins, or we should do the cleanup ourselves?